.sync.yml

```diff
@@ -3,6 +3,7 @@
   docker_sets:
     - set: centos6-64
     - set: centos7-64
+    - set: centos8-64
```

Comment:
This looks unrelated to strings

Reply:
That was missing at a pull request I merged earlier today.